test/verify/check-dashboard

```diff
@@ -94,100 +92,100 @@ class TestBasicDashboard(MachineCase, DashBoardHelpers):
         self.wait_dashboard_addresses (b2, [ "localhost" ])
         b.wait_present("#dashboard-hosts a[data-address='localhost'] button.pficon-delete.disabled")
 
-        self.add_machine (b, m2.address)
-        self.wait_dashboard_addresses (b, [ "localhost", m2.address ])
-        self.wait_dashboard_addresses (b2, [ "localhost", m2.address ])
+        self.add_machine (b, "10.111.113.2")
+        self.wait_dashboard_addresses (b, [ "localhost", "10.111.113.2" ])
+        self.wait_dashboard_addresses (b2, [ "localhost", "10.111.113.2" ])
```

Comment:
There were 10's if not 100 interpolation statements with addresses. These are all gone now, which on balance makes things much more readable and maintainable. The fixed IP addresses are now something we can rely on in tests. This is now similar to hard coding things like component names in URLs or HTML CSS selectors such as ```#id```.